Adyen/Analytics/AnalyticsProvider/TelemetryTracker.swift

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+/// : nodoc:
+public enum TelemetryFlavor {
+    case components(type: String)
+    case dropIn(type: String = "dropin", paymentMethods: [String])
+    case dropInComponent
```

Comment:
what's the difference between `dropIn` and `dropInComponent`? can we add a comment to explain the difference?

Reply:
Yes, definitely! `dropInComponent` are those components used within the DropIn. In those cases, we don't want to send the telemetry event since it's already being sent from the DropIn component with the different available payment methods.

`components` are stand alone components, outside of DropIn.

Reply:
Added a clarifying comment